cli/cmd/config.go

```diff
@@ -72,6 +72,12 @@ var resources = resourcemanager.NewRegistry().
 					{Header: "NAME", Path: "spec.name"},
 					{Header: "ENABLED", Path: "spec.enabled"},
 					{Header: "MINIMUM SCORE", Path: "spec.minimumScore"},
+					{Header: "PLUGINS", Path: "spec.total.plugins"},
```

Comment:
For the future: instead of showing just the number of plugins enabled, we could show the plugin names (or ids) separated by comma. What do you think?